awswrangler/catalog/_create.py

```diff
@@ -644,14 +644,42 @@ def create_database(
         args["Description"] = description
 
     try:
-        r = client_glue.get_database(Name=name)
-        if not exist_ok:
-            raise exceptions.AlreadyExists(f"Database {name} already exists and <exist_ok> is set to False.")
-        if description and description != r["Database"].get("Description", ""):
-            client_glue.update_database(**_catalog_id(catalog_id=catalog_id, Name=name, DatabaseInput=args))
+        client_glue.get_database(Name=name, CatalogId=catalog_id)
     except client_glue.exceptions.EntityNotFoundException:
+        database_exist = False
+    else:
+        database_exist = True
+
+    # 1st way
+    if database_exist and exist_ok and description and description != r["Database"].get("Description", ""):
```

Comment:
We check for description existence can be put also in a list like `[database_info["Database"].get("Description"), "", None]` to avoid repetition.

Reply:
Repetition of what?
Less code is not always better i think the condition as it is written is much more readable and understandable. The first condition tests if the database exists then if it is OK if it exists then if a description was given and then if the description is different. Also this is the correct order to evaluate from left to right, if any of them is False, Python will not evaluate the rest, in your case it will check on each list element first and at the end make a decision. 
For example: if no description is given why would you even check if it is empty or if it equal to the current Description.
I might even add for being explicit, `description is not None`. 
But is your PR, you can decide of course.